### DISCUSSION TITLE
Welcome to Tally-Connector Discussions!

### DISCUSSION THREAD

**Discussion by ramajayam-CA** [ramajayam-CA/Tally-Connector#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bharatone1** [comment]
Good afternoon, Sir,
The Connector.tcp expired on 31-12-2024. Could you please advise how we can continue using it?
